keps/sig-scheduling/785-graduate-component-config-to-beta/README.md

```diff
@@ -0,0 +1,110 @@
+# KEP-785: Graduate Scheduler Component Config API to Beta
```

Comment:
I think we should use this KEP to maintain the entire graduation lifecycle of scheduler ComponentConfig, instead of graduating it to a specific level.
If you agree, the resting wordings/sections need to adjust accordingly.

```suggestion
# KEP-785: Scheduler Component Config API
```

Reply:
Would that title imply to include the whole definition of the API in the KEP?